### ISSUE TITLE
Question re: RELEASES

### ISSUE THREAD

**Issue by PascalPixel** [Burzo/electron-forge-ssl-code-sign-plugin#3]
My updates are getting refused on Windows for an invalid signature, it’s been a week and I’m running out of ideas, so;
Is it possible that this plugin runs, changes the files (signed) and does not update the checksums in RELEASES file?
Thanks!

**Comment by Burzo** [comment]
Hey, sorry for the late reply. I assume you have problems with the `NuGet` packages right, those are used for OTA updates?

I noticed this exact bug with the eSigner tool and reported it to ssl.com months ago and it seems they fixed it recently in the v1.3.1 version. Try to use their new version, signing should work.

However, other then that, the RELEASES file isn't changed, the plugin just reads from it.

**Comment by PascalPixel** [comment]
ah that might be why it was fixed, i thought it was because i switched to this config instead;

```ts
const forgeConfig: ForgeConfig = {
  packagerConfig: {
    // invalid according to typescript, 
    // but this is the only place it works, 
    // all the other places you can put this key will *not* pick it up 
    // during the build process and use the built-in signTool
    // @ts-ignore
    signToolPath: path.resolve(
      "C:\\Users\\circleci\\project\\CodeSignTool.bat",
    ),
    signWithParams: [
      `-override="true"`,
      `-credential_id="${process.env.WINDOWS_CERT_CREDENTIAL_ID}"`,
      `-username="${process.env.WINDOWS_CERT_USER_NAME}"`,
      `-password="${process.env.WINDOWS_CERT_PASSWORD}"`,
      `-totp_secret="${process.env.WINDOWS_CERT_USER_TOTP}"`,
    ],
  },
};

export default forgeConfig;
```

oh well its all being signed now 😄
